### PR TITLE
Update observation details type

### PR DESCRIPTION
**Pull request by CantRoger**
Hi Vance,

Thanks for your help in getting the .ttl formatted right, looks much tidier now!
Think it's ready for review, let me know if there's any other steps before it can be brought in!

### CODE REVIEW

File: vocabularies/observation-detail-type.ttl [hunk 1]
```diff
@@ -1,44 +1,49 @@
...
 @prefix xml: <http://www.w3.org/XML/1998/namespace> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 <http://linked.data.gov.au/def/observation-detail-type> a owl:Ontology,
         skos:ConceptScheme ;
-    dct:created "2020-03-30"^^xsd:date ;
+    dct:created "2020-05-28"^^xsd:date ;
     dct:creator <http://linked.data.gov.au/org/gsq> ;
-    dct:modified "2020-03-30"^^xsd:date ;
+    dct:modified "2020-05-28"^^xsd:date ;
```
Comment (KellyVance) [inline vocabularies/observation-detail-type.ttl:17]: Just need to change the modified date to today

File: vocabularies/observation-detail-type.ttl [hunk 2]
```diff
@@ -1,44 +1,49 @@
...
+    skos:definition "Distance between readings or observations in metres"@en ;
     skos:inScheme <http://linked.data.gov.au/def/observation-detail-type> ;
-    skos:prefLabel "Example"@en ;
+    skos:prefLabel "Reading Interval (m)"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/observation-detail-type> .
 
-obsdt:placeholder a skos:Concept ;
-    skos:definition "This is a placeholder."@en ;
+obsdt:survey-configuration a skos:Concept ;
+    skos:definition "Addition configuration information about the survey configuration"@en ;
```
Comment (KellyVance) [inline vocabularies/observation-detail-type.ttl:39]:
```suggestion
    skos:definition "Additional information about the survey configuration"@en ;

```

**Comment by KellyVance** [comment]
@geoderekh can you please review and approve as the vocabulary owner.
As per usual, this is just a start. I'll need to make a bunch of changes in edition two once we start getting feedback on what is missing through migration and testing.

File: vocabularies/observation-detail-type.ttl (hunk 1 above)
Comment (KellyVance) [inline vocabularies/observation-detail-type.ttl:17]:
```suggestion
    dct:modified "2020-06-03"^^xsd:date ;

```

File: vocabularies/gsq-roles.ttl [hunk 3]
```diff
@@ -1,121 +1,131 @@
...
+    skos:definition "Agent that owns the resource"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles>;
+    skos:historyNote "Agent is known as Party in the original definition from ISO19115. The altLabels are drawn from P&G (general provisions) regulation 2017, section 37(4)"@en ;
+    skos:prefLabel "Owner"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles>.
+
+isorole:principalInvestigator a skos:Concept ;
+    skos:definition "Key party responsible for gathering information and conducting research"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles> ;
+    skos:altLabel "Observer"@en ;
```
Comment (geoderekh) [inline vocabularies/gsq-roles.ttl:105]: P.I.s and observers are not the same thing. Not even close. Even the definition says nothing about a party who observes. 
The altLabel facilitates a suggestion to use PI as the observation agent, which is wrong. the act of observing is independent of research and it needs to be pulled out as its own concept.

**Pull request merged**